metatron-app/metatron-discovery
# #2805 Improved overview screen on data exploration

### Description

1. If data is of type datasource, the first dashboard thumbnail is shown. (removed)
2. 'No data' notification label required when no data
3. The font size of the title is too large. (I think 22px seems reasonable) => it's a publishing issue.
4. Catalog panel changed
Caroussel UI
If there is no 1depth, no path should be indicated (In case of 'favorite catalog')
'All catalog' panel is activated on first entry

**Related Issue** : #2805 


### How Has This Been Tested?

1. Check if 'No Data' is displayed normally on the main screen.
2. Make sure the left panel is changed to Caroussel
3. Check if the font size of the title has changed

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
